tests/navigation/test_nav_mesh_bam.py

```diff
@@ -0,0 +1,165 @@
+from direct.showbase.ShowBase import ShowBase
+import direct.directbase.DirectStart
+from panda3d import navigation
+from panda3d import navmeshgen
+from panda3d import core
+from panda3d.core import NodePath
```

Comment:
Why import `NodePath` and `LPoint3` specifically when you grab everything else via `core`? Might as well keep the namespacing intact and use `core.NodePath` and `core.LPoint3`.